draft-davidson-pp-protocol.md

```diff
@@ -74,11 +74,36 @@ informative:
       -
         ins: F. Valsorda
         org: Independent
+  TrustTokenAPI:
```

Comment:
Probably pointing at the WICG repo as the canonical one makes sense, though maybe point it at the web.dev post until I get around to updating the WICG with the spec.

Reply:
I'll change it back, I think what is in the WICG repo works.